WPMedia/fusion-news-theme-blocks
# TBRANDS-51 - Layout alignment - Quilted Image Update

## Description

Switch quilted image block to not use position absolute to allow for better alignment with container sizing

## Jira Ticket

- [TBRANDS-51](https://arcpublishing.atlassian.net/browse/TBRANDS-51)

## Acceptance Criteria

## Test Steps

1. Checkout this branch `git checkout TBRANDS-51-layout-alignment`
2. Checkout out the Fusion branch - `TBRANDS-51-layout-alignment`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/quilted-image-block`
4. Visit homepage with blocks added and verify alignment 


## Dependencies or Side Effects

Requires Feature Pack Updates in PR = https://github.com/WPMedia/arc-themes-feature-pack/pull/233

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
